ddtrace/contrib/asgi/middleware.py

```diff
@@ -104,6 +119,8 @@ async def __call__(self, scope, receive, send):
             span_type=SpanTypes.WEB,
         )
 
+        scope["datadog_span"] = span
```

Comment:
Two suggestions: add a "datadog" dict we can use moving forward and specify the span as the "request" (aka root level span) span.

```suggestion
        scope["datadog"] = {"request_span": span}
```

Reply:
:+1:

Reply:
Done in #2726